src/Storage/Ephemeral.php

```diff
@@ -7,26 +7,28 @@
 
 class Ephemeral {
 
+	public $group_name;
```

Comment:
I assume the lack of dockblocks here will be addressed in a higher phpstan level? 👀

Reply:
ya, looks like probably level 3. sweet.